src/interpreter/ByteCodeGenerator.cpp

```diff
@@ -183,7 +185,7 @@ ByteCodeBlock* ByteCodeGenerator::generateByteCode(Context* context, Interpreted
     ByteCodeGenerateContext ctx(codeBlock, block, codeBlock->isGlobalScope(), codeBlock->isEvalCode(), inWithFromRuntime || codeBlock->inWith(), nData);
 
 #ifdef ESCARGOT_DEBUGGER
-    ByteCodeBreakpointContext breakpointContext;
+    ByteCodeBreakpointContext breakpointContext(context->debugger()->parsingEnabled());
```

Comment:
At this point, we could ensure that context->debugger() has valid value?
If so, check code located in the same function below seems unnecessary.
```c++
if (context->debugger() && context->debugger()->enabled() && breakpointContext.m_parsingEnabled)
```